### PR TITLE
chore(deps)!: upgrade consola to v3 and minimum node version 

### PR DESCRIPTION
**Pull request by rchl**
### 🔗 Linked issue

https://github.com/nuxt/nuxt/issues/24343

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Addressing the https://github.com/nuxt/nuxt/issues/24343 issue which is about outdated `consola` being used in a couple of Nuxt 2 packages.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by rchl** [comment]
Bumped node in CI to 16. Maybe it will fix it.

The package specifies 12.22.12 as minimum version in `package.json` which is unreasonable these days anyway but I have not bumped that to avoid breaking changes (I'm assuming that since the code is built, it might still work with node 12.x)

File: package.json [hunk 1]
```diff
@@ -50,6 +50,6 @@
     "webpack": "3 || 4 || 5"
   },
   "engines": {
-    "node": ">=v12.22.12"
+    "node": ">=14.18.0"
```
Comment (pi0) [inline package.json:53]: (we might upgrade to higher version in next major, will have to check other deps)

File: src/reporters/basic.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { consola } from '../utils/cli'
-import { Reporter } from 'src/types'
+import { Reporter } from '../types'
```
Comment (pi0) [inline src/reporters/basic.ts:2]: Was this change necessary?

Comment (rchl) [reply src/reporters/basic.ts:2]: Caused error in the editor:

Screenshot 2023-11-17 at 11 40 45

If you insist then I can look into fixing that (probably a missing mapping or `baseDir` in `tsconfig` but I though it's better to be consistent and use paths relative to the current file anyway.

Comment (rchl) [reply src/reporters/basic.ts:2]: Also this change doesn't affect compiled bundle, as far as I can tell, so IMO keeping this change is better.

Comment (pi0) [reply src/reporters/basic.ts:2]: Yeah it is better (hate old me that didn't used relative haha) only preferred that if it was not part of this commit but also okay 👍🏼

**Comment by pi0** [comment]
@rchl I might need to delay this release to next week as have some priorities. If it is a blocker for something, please mention.

**Comment by rchl** [comment]
No, not a blocker. Thanks.
